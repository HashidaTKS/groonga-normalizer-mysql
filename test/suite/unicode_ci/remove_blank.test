register normalizers/mysql

normalize NormalizerMySQLUnicodeCI "a b c" REMOVE_BLANK
