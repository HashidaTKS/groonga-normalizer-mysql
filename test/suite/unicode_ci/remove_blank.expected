register normalizers/mysql
[[0,0.0,0.0],true]
normalize NormalizerMySQLUnicodeCI "a b c" REMOVE_BLANK
[[0,0.0,0.0],{"normalized":"ABC","types":[],"checks":[]}]
